source/playground.liquid

```diff
@@ -21,7 +21,20 @@ is_playground: true
         data-tabbar="item"
         class="sl-c-button sl-c-button--tab"
         data-setting="inputFormat">Sass</button>
-      <span data-tabbar="item" class="sl-c-playground__tabbar-title tabbar-title sl-c-playground__tabbar-version"></span>
+        <span data-tabbar="item" class="sl-c-playground__tabbar-title tabbar-title sl-c-playground__tabbar-version"></span>
+        <button
+          type="button"
+          id="playground-copy-input"
+          class="sl-c-button sl-c-button--copy sl-c-button--icon">
+          <div class="sl-c-playground-button--copy--copy">
+            {%- render 'icons/copy' -%}
+            <span class="visuallyhidden">Select to Copy</span>
+          </div>
+          <div class="sl-c-playground-button--copy--copied">
+            {%- render 'icons/circle-check' -%}
+            <span class="visuallyhidden">Code is Copied</span>
```

Comment:
When using the screen reader, it was really hard to distinguish between the Select to "Copy" and the word "Copied" that is read right after. They sounded the exact same to me so I added the "Code is" before the word "Copied".